### PR TITLE
feat: 3rd register page

### PR DESCRIPTION
**Pull request by tokitouq**

### CODE REVIEW

File: backend/django_core/templates/user/register/_3.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<div class="flex flex-col gap-2 whitespace-nowrap font-bold uppercase leading-none tracking-widest text-white md:text-[1.2vw] md:gap-[0.5vw]">
+    <span class="flex items-center text-base font-bold uppercase tracking-widest text-white md:text-[1.2vw]">
+        welcome to&nbsp;
+        <p class="inline-flex">
+            <span class="text-white/90">c</span>
+            <span class="text-warning">o</span>
+            <span class="text-white/90">r</span>
+            <span class="text-white/90">e</span>
```
Comment (baseplate-admin) [inline backend/django_core/templates/user/register/_3.html:8]: Export to component

File: backend/django_core/templates/user/register/_3.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    </div>
+</div>
+<div class="flex items-center justify-between">
+    <div class="flex flex-col gap-1 md:gap-[0.5vw]">
+        <span class="text-xs leading-none text-surface-100 md:text-[0.75vw]">Already have an account?</span>
+        <a
+            href="{% url 'login_view'%}"
+            class="text-base leading-none md:text-[1.1vw] text-primary underline"
+        >
+            Login
```
Comment (baseplate-admin) [inline backend/django_core/templates/user/register/_3.html:42]: Should this be login?

**Review by baseplate-admin — CHANGES_REQUESTED** [review]
Take a look

File: backend/django_core/templates/user/register/_3.html (hunk 2 above)
Comment (tokitouq) [reply backend/django_core/templates/user/register/_3.html:42]: Yes? no?
its register page so login? no? :confused:

File: backend/django_core/apps/pages/views/user.py [hunk 3]
```diff
@@ -160,10 +160,19 @@ def register_view(request: HttpRequest) -> HttpResponse:
                     form.fields["username"].widget.attrs["class"] += " focus:border-error"
 
             return render(request, "user/register/_2.html", context={"form": form})
-        else:
+        elif _internal_state_ == 3:
             form_data = request.session["_form_"]
-            print(form_data)
-            return HttpResponse(200)
+            username = form_data["username"][0] if form_data.get("username") else None
+            email = form_data["email"][0] if form_data.get("email") else None
```
Comment (baseplate-admin) [inline backend/django_core/apps/pages/views/user.py:166]: What in god's name is this?

Comment (tokitouq) [reply backend/django_core/apps/pages/views/user.py]: Ehh you dont get it?
its a simple logic, even my doggy can understand :sweat_smile:

**Pull request merged**